language/src/main.rs

```diff
@@ -443,11 +449,10 @@ fn handle_crate<'tcx>(
                             .as_str(),
                     ))
                 }
-                "v" => {
+                "v" | "v_ssprove_state" | "v_ssprove_pure" | "v_ssprove" => {
```

Comment:
Please add these to the help message of the command and explain a little the differences.